bigtable/client/cell.h

```diff
@@ -64,6 +76,12 @@ class Cell {
   /// this object is deleted.
   std::string const& value() const { return value_; }
 
+  /// Return the contents of this cell. The returned value is not valid after
```

Comment:
The comment is not accurate: (a) it returns the current value as a numeric, and (b) the return value is valid after the object is deleted (because it is not returned by reference).